bin/deploy

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
```

Comment:
Glad to see this getting used :) It probably makes sense to move it to a gem sooner rather than later if we're going to reuse it.